core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.ts

```diff
@@ -207,7 +207,7 @@ export class DotContentEditorComponent implements OnInit, OnChanges {
                     if (!this.checkIfAlreadyExists(contentType)) {
                         this.getcontainerStructures.push(
                             new FormGroup({
-                                code: new FormControl('', [Validators.required]),
+                                code: new FormControl(''),
```

Comment:
Can we test this to make sure no one marks it as `required`?

Reply:
Noted

Reply:
I removed this function because it is not use anywhere in the code so don't need to test here.